clinica/utils/freesurfer.py

```diff
@@ -195,28 +195,28 @@ def extract_image_id_from_longitudinal_segmentation(freesurfer_id):
 
     Example:
         >>> from clinica.utils.freesurfer import extract_image_id_from_longitudinal_segmentation
-        >>> extract_image_id_from_longitudinal_segmentation('sub-CLNC01_ses-M00')
-        image_id(participant_id='sub-CLNC01', session_id='ses-M00', long_id='')
+        >>> extract_image_id_from_longitudinal_segmentation('sub-CLNC01_ses-M000')
+        image_id(participant_id='sub-CLNC01', session_id='ses-M000', long_id='')
         >>> extract_image_id_from_longitudinal_segmentation('sub-CLNC01_long-M0018')
```

Comment:
Should the longitudinal IDs be changed here too?

Reply:
yup they should !